easybuild/easyconfigs/q/Qt5/Qt5-5.13.1-GCCcore-8.3.0.eb

```diff
@@ -34,7 +34,8 @@ builddependencies = [
     ('gperf', '3.1'),
     ('Ninja', '1.9.0'),
     ('Python', '3.7.4'),
-    ('re2c', '1.2.1')
+    ('re2c', '1.2.1'),
+    ('JasPer', '2.0.14'),
```

Comment:
@terjekv You shouldn't add `JasPer` both as build dep and runtime dep.

If a dependency is listed in `dependencies`, it will already be loaded at build time too.

This is also why the test fails, you're adding `JasPer` twice in the same dependency graph.